StantUiIosLibrary/Classes/Tables/GroupSelectionTableViewWithCollapsedSearchBar/TableView/Cell/GroupSelectionRowCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  GroupSelectionRowCell.swift
+//  StantUiIosLibrary
+//
+//  Created by Mac Mini Novo on 30/07/19.
+//
+import UIKit
+
+public class GroupSelectionRowCell: UITableViewCell {
+    
+    internal static let IDENTIFIER          = "GroupSelectionRowCellIdentifier"
```

Comment:
Just saw I had an idea: You could create a extension for the general type (Like class, or object, whatever...) and have a method like this:  https://www.natashatherobot.com/nsstringfromclass-in-swift/ :)

Reply:
Nice suggestion! I created a function on UIView extension and already changed on all identifiers. :)